vms/platformvm/txs/executor/staker_tx_verification.go

```diff
@@ -165,11 +180,13 @@ func verifyAddValidatorTx(
 		return nil, fmt.Errorf("%w: %w", ErrFlowCheckFailed, err)
 	}
 
-	// Make sure the tx doesn't start too far in the future. This is done last
-	// to allow the verifier visitor to explicitly check for this error.
-	maxStartTime := currentTimestamp.Add(MaxFutureStartTime)
-	if startTime.After(maxStartTime) {
-		return nil, ErrFutureStakeTime
+	if !isDurangoActive {
+		// Make sure the tx doesn't start too far in the future. This is done last
+		// to allow the verifier visitor to explicitly check for this error.
+		maxStartTime := currentTimestamp.Add(MaxFutureStartTime)
+		if preDurangoStartTime.After(maxStartTime) {
+			return nil, ErrFutureStakeTime
```

Comment:
we must guarantee this is the last error that occurs, if an error occurs.
Should be possible to cleanup post fork actiation